tech/languages/nodejs/modules.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Modules
+page: nodejs
+order: 2
+---
+
+# Modules
+
+## Installation
+
+To install additional modules from Fedora repositories, use:
+
+```
+$ sudo dnf install nodejs-<module-name>
+```
+
+or 
+
+```
+$ sudo dnf install 'npm(module-name)'
+```
+
+For example, to install express, you need to type:
+
+```
+$ sudo dnf install nodejs-express
+```
+
+or 
+
+```
+$ sudo dnf install 'npm(express)'
+```
+
+Most modules are prefixed with 'nodejs-'. There are, however, a few exceptions (for example mocha), which are not prefixed.
+
+## Using modules
+
+To be able to require() installed modules, you must first explicitly link them.
+
+```
+$ sudo link express
+```
+
+If you really want this behavior, you can set $NODE_PATH as follows: 
```

Comment:
What behavior? To get global modules by default, or to make linked modules work?